package.json

```diff
@@ -17,9 +17,11 @@
   },
   "homepage": "https://github.com/sashkivna/viewRendering#readme",
   "dependencies": {
+    "@uirouter/angularjs": "^1.0.20",
     "angular": "^1.7.4",
     "angularjs": "0.0.1",
-    "jquery": "^3.3.1"
+    "jquery": "^3.3.1",
+    "ngstorage": "^0.3.11"
```

Comment:
what is `ngstorage`?

Reply:
debugger removed.
deleted ngStorage from package.json